cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1204,8 +1204,11 @@ def default_for(
                    The workflow polls for task status (no task messaging)
                 ssh
                    Use non-interactive ssh for task communications
+                https
+                   Via the running Hub proxy and/or UI-Server (requires UI
+                   Server installation)
 
-                .. versionchanged:: 8.0.0
+                .. versionchanged:: 8.2.0
```

Comment:
This version changed annotation was because the config had moved.

```suggestion
                .. versionchanged:: 8.0.0
```